Bottom: 1282f9b7e93e40d27ca599d86df8f49a2f38cf9c
Top:    feeca4a2212b06066ce62c3bfb65d193c52e0a4a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:45 +0100

Fix for PCB+GL Experimental


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 52e894d..295e656 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -854,9 +854,9 @@ do_hole (const BoxType *b, void *cl)
 }
 
 static bool
-polygon_contains_user_holes (PolygonType *polygon)
+pour_contains_user_holes (PourType *pour)
 {
-  return (polygon->HoleIndexN > 0);
+  return (pour->HoleIndexN > 0);
 }
 
 
@@ -890,7 +890,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box /*, bool forc
   /* Polygon has holes.. does it have any user-drawn holes?
    * If so, it must be masked with a _new_ stencil bit.
    */
-  use_new_stencil = force_new_stencil | polygon_contains_user_holes (poly);
+  use_new_stencil = force_new_stencil || true; // | polygon_contains_user_holes (poly);
 
   if (use_new_stencil)
     {
